packages/atomic/src/global/environment.ts

```diff
@@ -0,0 +1,13 @@
+import {VERSION} from '@coveo/headless';
+
+export interface AtomicEnvironment {
+  version: string;
+  headlessVersion: string;
+}
+
+export default function () {
```

Comment:
This function is called by Stencil when the script is loaded in the browser.

Reply:
This would be a useful information to have in the file about the function. Could we add it as a comment?